fe/src/pages/index.vue

```diff
@@ -29,7 +27,55 @@
         <IconsBlocksRight />
       </div>
     </div>
+    <div class="bg-primary-50">
+      <div class="pt-16 mx-44 pb-24">
+        <div
+          class="bg-gradient text-transparent bg-clip-text text-2xl font-bold leading-[125%]"
+        >
+          All Campaigns
+        </div>
+        <div
+          class="mt-6 grid grid-cols-3 gap-6 sm:grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 campaigns"
+        >
+          <base-campaign-card
+            v-for="i in 9"
+            :key="i"
+            :title="cardValueSample.title"
+            :description="cardValueSample.description"
+            :eth-value="cardValueSample.ethValue"
+            :img-src="cardValueSample.imgSrc"
+            :days-left="cardValueSample.daysLeft"
```

Comment:
We can destructure the cardValueSample so that we don't have to keep using `cardValueSample.__`.
```suggestion
            :title="title"
            :description="description"
            :eth-value="ethValue"
            :img-src="imgSrc"
            :days-left="daysLeft"
```